modules/dailyhuntBidAdapter.js

```diff
@@ -0,0 +1,193 @@
+import { registerBidder } from '../src/adapters/bidderFactory';
+import * as utils from '../src/utils';
+
+const BIDDER_CODE = 'dailyhunt';
+const BIDDER_ALIAS = 'dh';
+const SUPPORTED_MEDIA_TYPES = ['banner', 'native'];
+
+const PROD_PREBID_ENDPOINT_URL = 'https://money.dailyhunt.in/openrtb2/auction';
+
+const PROD_ENDPOINT_URL = 'https://money.dailyhunt.in/openx/ads/index.php';
+
+function buildParams(bid) {
+  let params = { ...bid.params };
+  params.pagetype = 'sources';
+  params.placementId = 12345;
+  params.env = 'prod';
+  if (params.testmode && params.testmode === true) {
+    params.customEvent = 'pb-testmode';
+  }
+  let hasWeb5Size = false;
+  let hasWeb3Size = false;
+  bid && bid.sizes && bid.sizes.forEach((size, i) => {
+    if (!hasWeb3Size && size[0] == 300 && size[1] == 250) {
+      hasWeb3Size = true;
+    }
+    if (!hasWeb5Size && (size[0] == 300 || size[0] == 320) && size[1] == 50) {
+      hasWeb5Size = true;
+    }
+  })
+  params.zone = 'web';
+  if (hasWeb3Size && hasWeb5Size) {
```

Comment:
Can this be simplified?

Suggestion: if hasWeb5Size and not hasWeb3Size then 'web-5', else 'web-3'

Reply:
Changes Fixed.